src/services/isProductEpired.service.js

```diff
@@ -0,0 +1,27 @@
+import moment from 'moment'
+import {Products} from '../database/models'
+import { Emitter } from '../events/emmitEvent'
+import { Sequelize } from 'sequelize'
+
+export const ExpiredProducts=async()=>{
+    try{
+    const isProductExpired=await Products.findAll({where:{
+        expiryDate:{
+            [Sequelize.Op.lt]: moment().subtract(process.env.EXPIRESIN,process.env.TIMEUNITS)
+          }
+    }})
+          console.log(isProductExpired)
+    if (isProductExpired){
```

Comment:
on this **isProductExpired** variable,  Make sure to handle the case when this **isProductExpired** is an empty array. In this implementation, it will still update the isExpired flag and emit the event even if no products have expired. :red_circle:

Reply:
i've made the changes you asked sir